omod/src/main/webapp/resources/htmlforms/mexico/vitals.xml

```diff
@@ -519,20 +268,6 @@
             <legend><uimessage code="mirebalais.vitals.temperature.title"/></legend>
             <h3><uimessage code="mirebalais.vitals.temperature.title"/></h3>
 
-            <!-- only show/handle the fahrenheit field if we are in 'ENTER' or 'EDIT' mode -->
-            <ifMode mode="VIEW" include="false">
-                <p class='left'>
-                    <span class='obs-field'>
-                        <input type="text" id="temperature_f" size="5" min="77.0" max="109" class="number numeric-range fahrenheit"/>
-                        <span class="append-to-value"><uimessage code="emr.units.degreesFahrenheit"/></span>
-                        <span class="error field-error" style="display: none"></span>
-                    </span>
-                </p>
-                <p class="left">
-                    <uimessage code="mirebalais.vitals.or"/>
-                </p>
-            </ifMode>
-
```

Comment:
I didn't review this too deeply, but it looks good, I'll trust you pulled the right code... (or at least will test to make sure it works).

Nice to get rid of this, if at least in Mexico.  Story behind this... when we opened Mirebalais we all agreed to standardize on metric, and then when Renee (former MedInfo member) got to Mirebalais to do training she discovered that of the donated scales, some measured in metric and other in imperial...